packages/addons/paraglide/index.ts

```diff
@@ -204,8 +164,7 @@ export default defineAddon({
 			}
 			htmlNode.attribs = {
 				...htmlNode.attribs,
-				lang: '%paraglide.lang%',
-				dir: '%paraglide.textDirection%'
```

Comment:
removing textDir because users reported issues with the [Intl API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getTextInfo) behaving differently in different runtimes. 

it's better to have a working setup for everyone that users can improve themselves than a half broken setup